ext/.gitignore

```diff
@@ -0,0 +1,4 @@
+**/*
```

Comment:
Since we rm -rf this anyway during a build, what's the point of adding a `.gitignore` at this level? Shouldn't we add the `.gitignore` rules on the top-level `.gitignore`?